app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -839,6 +839,7 @@ public static boolean isDataStale() {
                 .where("timestamp <= " + endTime)
                 .where("calculated_value != 0")
                 .where("raw_data != 0")
+                .where("calibration_uuid != \"\"")
```

Comment:
this method is only used for syncing with wear is that right?

Reply:
Yes.   Ready to merge.